include/mbedtls/dhm.h

```diff
@@ -71,9 +81,18 @@
     "E39E772C180E86039B2783A2EC07A28FB5C55DF06F4C52C9" \
     "DE2BCBF6955817183995497CEA956AE515D2261898FA0510" \
     "15728E5A8AACAA68FFFFFFFFFFFFFFFF"
-
+/** 
+ * The generator of the 2048-bit MODP Group, as defined in <em>RFC-3526: 
```

Comment:
Blocker as above, suggestion: `The chosen generator of the 2048-bit group from RFC 3526`.

Reply:
rephrased